src/utils/base.ts

```
@@ -3,13 +3,14 @@ import { LRUCache } from "lru-cache"
 
 import { PrismaClient } from "../generated/prisma/client.js"
 
-import type { Prisma } from "../generated/prisma/client.js"
+import { Prisma } from "../generated/prisma/client.js"
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(`@typescript-eslint/consistent-type-imports`)

---

[error] Unable to resolve path to module '../generated/prisma/client.js'.

(import-x/no-unresolved)